jenkins/catapult/baseImageDockerfileGenerator.py

```diff
@@ -355,8 +358,26 @@ def add_git_dependency(organization, project, versions_map, options, revision=1)
 	], ORGANIZATION=organization, PROJECT=project, VERSION=version, OPTIONS=' '.join(options), REVISION=revision)
 
 
+def add_openssl(options, configure):
+	version = options.versions['openssl_openssl']
+	compiler = 'linux-x86_64-clang' if options.is_clang else 'linux-x86_64'
+	print_line([
+		'RUN git clone https://github.com/openssl/openssl.git',
+		'cd openssl',
+		'git checkout {VERSION}',
+		'{OPEN_SSL_OPTIONS} perl ./Configure {COMPILER} {OPEN_SSL_CONFIGURE} --prefix=/usr/local --openssldir=/usr/local',
+		'make -j 8',
+		'make install',
+		'cd ..',
+		'rm -rf openssl'
+	], OPEN_SSL_OPTIONS=' '.join(options.openssl()), OPEN_SSL_CONFIGURE=' '.join(configure), VERSION=version, COMPILER=compiler)
+
+
 def generate_phase_deps(options):
 	print(f'FROM {options.layer_image_name("boost")}')
+
+	add_openssl(options, [])
```

Comment:
why are you always installing openssl now ?

Reply:
Ubuntu got upgraded from 20.04 -> 22.04.  22.04 ships with openssl 3.x and catapult is still using openssl 1.1.1 so need to install in the base image